docs/source/common/lightning_module.rst

```diff
@@ -932,9 +932,93 @@ True if using TPUs
 
 --------------
 
+automatic_optimization
```

Comment:
I am confused. Shouldn't it be manual optimisation ?

Reply:
the property is called `automatic_optimization`

Reply:
@tchaton ^^